src/SearchResults.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import moment from "moment";
+moment().format();
+
+function SearchResults(props) {
+  const table = (
+    <table className="table">
+      <thead>
+        <tr>
+          <th scope="col">id</th>
+          <th scope="col">Title</th>
+          <th scope="col">First Name</th>
+          <th scope="col">Surname</th>
+          <th scope="col">Email</th>
+          <th scope="col">Room id</th>
+          <th scope="col">Check In Date</th>
+          <th scope="col">Check Out Date</th>
+          <th scope="col">Number of Nights</th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.results.map(result => {
+          let arrivalDate = moment(result.checkInDate, "YYYY-MM-DD");
```

Comment:
```suggestion
          const arrivalDate = moment(result.checkInDate, "YYYY-MM-DD");
```

Reply:
I would consider defaulting to use the `const` keyword here as you don't intend on re-assigning these variables.
But it isn't a huge deal 😄